serviceWrappers/linux/kLiveController

```diff
@@ -57,7 +57,7 @@ loadNvm() {
 
 start() {
     [ -d ${LOG_DIR} ] || mkdir -p ${LOG_DIR}
-    loadNvm
+    #loadNvm
```

Comment:
@kalturaguy @yossipapi @GElkayam - the official way to install nodejs is by using their official rpm and deb packages.
Debina/Ubuntu: https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions
RedHat/Fedora: https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora
As a design guideline, when creating rpm/deb packages it is best practice to use other rpm/deb packages whenever available as dependancies. That allows us to continue using the same system for tracking changes, dependancies, and overall provide a more reliable way to install the platform.
NVM requires further building of non rpm/deb packages, it requires python, and overall introduces further dependancies that will complicate the platform deployment and make the system prone for other installation errors that can't leverage the standard rpm/deb install mechanism.

For SaaS we manage the environment, so there's very little risk. But when deployed and managed by others on other environment, we need the install packages to be as fail safe as possible, and thus minimizing the list of dependancies is critical.

Thanks!